pkg/org/auth.go

```diff
@@ -145,37 +152,43 @@ func ProjectFromRequest(app *bunapp.App, req bunrouter.Request) (*bunconf.Projec
 var jwtSigningMethod = jwt.SigningMethodHS256
 
 func encodeUserToken(app *bunapp.App, userID uint64, ttl time.Duration) (string, error) {
-	claims := &jwt.StandardClaims{
+	claims := &jwt.RegisteredClaims{
 		Subject:   strconv.FormatUint(userID, 10),
-		ExpiresAt: time.Now().Add(ttl).Unix(),
+		ExpiresAt: jwt.NewNumericDate(time.Now().Add(ttl)),
 	}
 	token := jwt.NewWithClaims(jwtSigningMethod, claims)
 
 	key := []byte(app.Config().SecretKey)
 	return token.SignedString(key)
 }
 
-func decodeUserToken(app *bunapp.App, jwtToken string) (uint64, error) {
-	token, err := jwt.ParseWithClaims(jwtToken, &jwt.StandardClaims{},
+func decodeUserToken(app *bunapp.App, jwtToken string) (uint64, string, error) {
```

Comment:
We probably want to return a `*bunconf.User` from here so it is easier to add more info to the JWT token.

But this is fine for now.